babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -874,6 +874,26 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testCreateTableDateFormatDateOnlyColumnLevelAttributes() {
+    final String sql = "create table foo (bar date)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` DATE)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testCreateTableDateFormatWithFormatStringColumnLevelAttributes() {
```

Comment:
nit: Attributes -> Attribute

Reply:
Here also, I would suggest renaming this to something similar to:

testCreateTableWithDateFormatStringColumnLevelAttribute()

Reply:
Thanks, fixed in https://github.com/googleinterns/calcite/pull/103/commits/759efab9843a94cc2467c3e2d47333e5b1c7ef5a.